pkg/telemetry/client_test.go

```diff
@@ -127,14 +129,21 @@ func TestNewClient_PacketSizeIsZero(t *testing.T) {
 }
 
 func TestRun_ReportNoCalled(t *testing.T) {
-	client := &MockClient{}
+	client, err := NewClient("127.0.0.1:0", ClientOpts{
+		Application: "qwe",
+		Instance:    "",
+		Interval:    time.Millisecond,
+		PacketSize:  0,
+	})
+	assert.NoError(t, err)
 
-	ctx := &MockContext{}
+	client.send = func(context.Context, admmonkit.Options) error {
```

Comment:
This is still a mock :-)
But, yeah, much tinier.

Reply:
Yeah, I know. The test itself isn't that great to begin with.